test/Icicle/Test/Sea/Seaworthy.hs

```diff
@@ -34,7 +33,7 @@ namer = A.namerText (flip Var 0)
 --   Like prop_psv, but without psv.
 prop_seaworthy wt
  = testIO
- $ do let seaProgram = Map.singleton (Attribute "eval") (wtAvalanche wt)
+ $ do let seaProgram = Map.singleton (Attribute "eval") (wtAvalancheFlat wt)
```

Comment:
huh?! it was trying to evaluate the unflattened program? oh, or did you just rename the field?